src/App.js

```diff
@@ -18,8 +20,12 @@ const Home = React.forwardRef((props, ref) => {
     <>
       <MainBody ref={ref} />
       <AboutMe />
-      <Project />
-      {showSkills && <Skills />}
+	  {/*<ProjectTimeline />*/}
```

Comment:
@bibinvargheset Please remove the commented code. I'm assuming that we don't need it in future.